marimo/_plugins/core/media.py

```diff
@@ -39,9 +39,7 @@ def io_to_data_url(
     if src is None:
         return None
 
-    if isinstance(src, io.BytesIO):
-        # clone before reading, so we don't consume the stream
-        src = io.BytesIO(src.getvalue())
```

Comment:
do we still want this? otherwise we can only read it once

Reply:
Added a tell() and seek back to original position (files don't support `getvalue()`).